packages/fetchai/agents/confirmation_aea_aw5/aea-config.yaml

```diff
@@ -0,0 +1,159 @@
+agent_name: confirmation_aea_aw5
+author: fetchai
+version: 0.1.0
+license: Apache-2.0
+description: This agent purchases information from other agents as specified in its
+  configuration. It acts as the confirmation AEA in Agent World 5.
+aea_version: '>=1.0.0, <2.0.0'
+fingerprint: {}
+fingerprint_ignore_patterns: []
+connections:
+- fetchai/http_client:0.22.0
+- fetchai/ledger:0.18.0
+- fetchai/p2p_libp2p:0.21.0
+- fetchai/soef:0.22.0
+contracts: []
+protocols:
+- fetchai/contract_api:1.0.0
+- fetchai/default:1.0.0
+- fetchai/fipa:1.0.0
+- fetchai/http:1.0.0
+- fetchai/ledger_api:1.0.0
+- fetchai/oef_search:1.0.0
+- fetchai/signing:1.0.0
+- fetchai/state_update:1.0.0
+skills:
+- fetchai/confirmation_aw3:0.10.0
+- fetchai/generic_buyer:0.25.0
+default_ledger: fetchai
+required_ledgers:
+- fetchai
+default_routing:
+  fetchai/http:1.0.0: fetchai/http_client:0.22.0
+  fetchai/ledger_api:1.0.0: fetchai/ledger:0.18.0
+  fetchai/oef_search:1.0.0: fetchai/soef:0.22.0
+connection_private_key_paths: {}
+private_key_paths: {}
+logging_config:
+  disable_existing_loggers: false
+  version: 1
+  formatters:
+    standard:
+      format: '[%(levelname)s]: %(message)s'
+    extra:
+      format: '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+  handlers:
+    logfile:
+      class: logging.FileHandler
+      formatter: extra
+      level: NOTSET
+      filename: all.log
+    logfile_error:
+      class: logging.FileHandler
+      formatter: extra
+      level: ERROR
+      filename: error.log
+    console:
+      class: logging.StreamHandler
+      formatter: standard
+      level: INFO
+  loggers:
+    aea:
+      handlers:
+      - logfile
+      - logfile_error
+      - console
+      level: DEBUG
+      propagate: false
+dependencies:
+  aea-ledger-fetchai:
+    version: <2.0.0,>=1.0.0
+default_connection: fetchai/p2p_libp2p:0.21.0
+---
+public_id: fetchai/p2p_libp2p:0.21.0
+type: connection
+config:
+  delegate_uri: null
+  public_uri: null
+  entry_peers:
+  - /dns4/acn.fetch.ai/tcp/9001/p2p/16Uiu2HAmVWnopQAqq4pniYLw44VRvYxBUoRHqjz1Hh2SoCyjbyRW
+---
+public_id: fetchai/soef:0.22.0
+type: connection
+config:
+  chain_identifier: fetchai_v2_testnet_incentivised
+  token_storage_path: /data/soef_key.txt
+---
+public_id: fetchai/ledger:0.18.0
+type: connection
+config:
+  ledger_apis:
+    ethereum:
+      address: null
+    fetchai:
+      address: https://rest-agentworld.prod.fetch-ai.com:443
+      chain_id: agentworld-1
+---
+public_id: fetchai/confirmation_aw3:0.10.0
+type: skill
+behaviours:
+  search:
+    args:
+      search_interval: 900
+models:
+  registration_db:
+    args:
+      custom_path: /data/registration.db
+  strategy:
+    args:
+      aw1_aea: null
+      leaderboard_token: null
+      leaderboard_url: null
+      registration_required: false
```

Comment:
Easiest option seemed to be this flag which essentially voids the existing checks in `is_valid_counterparty` below.

Reply:
> Easiest option seemed to be this flag which essentially voids the existing checks in `is_valid_counterparty` in `confirmation_aw3`'s strategy below.

It's a bit more complicated. Check out `register_counterparty`. This was previously called when a message from `confirmation_aw1` arrived. Which raises the question when we call this now. Because somehow we need to get the developer handle. Back to the drawing board.